tests/environments/storage/test_docker_storage.py

```diff
@@ -13,6 +13,11 @@
 from prefect.environments.storage import Docker
 
 
+@pytest.fixture
+def no_docker_host_var(monkeypatch):
+    monkeypatch.delenv("DOCKER_HOST", raising=False)
```

Comment:
Can you add a docstring to this fixture to mention its purpose

Reply:
gladly.  does that work?